R/validate_inputs.R

```diff
@@ -152,7 +152,7 @@ gather_messages <- function(iv) {
     failing_inputs <- Filter(Negate(is.null), status)
     unique(lapply(failing_inputs, function(x) x[["message"]]))
   } else {
-    logger::log_warn("Validator is disabled and will be omitted.")
```

Comment:
This was intended to explicitly send this message to the logger rather than the console. The respective unit tests accounted for that.

Reply:
Yes I noticed this and I feel it's not very correct. 

If you put an expectation on the string, you are actually testing logger functionality. It's also dependent on logger threshold, layout formatter, appender function etc, e.g. `"\\[WARN\\].+Validator is disabled and will be omitted."` instead of just `"Validator is disabled and will be omitted."`.
Our tests should be more independent on logger OR if we explicitly test logger then we should pre-set all of those options (testthat::with_option).
When thinking about this I came across a promising logger functionality that could address this issue nicely. I opened a separate issue for a wider discussion: https://github.com/insightsengineering/teal/issues/1028